src/primitives/VirtualList.tsx

```diff
@@ -0,0 +1,137 @@
+import * as s from 'solid-js';
+import * as lng from '@lightningtv/solid';
+import * as lngp from '@lightningtv/solid/primitives';
+import { List } from '@solid-primitives/list';
+import * as utils from '../utils.js';
+
+const rowOnLeft = lngp.handleNavigation('left');
+const rowOnRight = lngp.handleNavigation('right');
+const rowScroll = lngp.withScrolling(true);
+
+const rowStyles: lng.NodeStyles = {
+  display: 'flex',
+  gap: 30,
```

Comment:
Can we drop the default `gap` width of `30`?
Every user will have a different design system and different gaps for different components.
There is no point in trying to guess the default value for everyone.
Default gap of `0` is the most expected, as this is what you get with `display: "flex"`, you change that el to `Row` and suddenly you have some random gap.